homeassistant/components/nina/const.py

```diff
@@ -16,6 +16,9 @@
 CONF_FILTER_CORONA: str = "corona_filter"
 
 ATTR_HEADLINE: str = "Headline"
+ATTR_DESCRIPTION: str = "Description"
```

Comment:
State attribute keys should be lowercase snake_case.

Reply:
Addressed in #68911